tests/codegen/common.h

```diff
@@ -248,13 +260,30 @@ static std::string rewrite_attrs(const std::string &line)
 #endif
 }
 
-static std::string rewrite_local_unnamed_addr(const std::string &line)
+static std::string rewrite_function_hdr(const std::string &line)
 {
 #if LLVM_VERSION_MAJOR >= 10
-  // FROM: define i64 @BEGIN(i8*) local_unnamed_addr section \"s_BEGIN_1\" {
-  // TO:   define i64 @BEGIN(i8* %0) local_unnamed_addr section \"s_BEGIN_1\" {
-  static std::regex re("(@[^\\(]+)\\(([^\\)]+)\\)");
-  return std::regex_replace(line, re, "$1($2 %0)");
+  // FROM: define internal i64 @linear(i64, i64, i64, i64) #1 section
+  // \"helpers\" { TO:   define internal i64 @linear(i64 %0, i64 %1, i64 %2, i64
+  // %3) #1 section \"helpers\" {
```

Comment:
Any reason not to newline like this?
```suggestion
  // FROM: define internal i64 @linear(i64, i64, i64, i64) #1 section \"helpers\" {
  // TO:   define internal i64 @linear(i64 %0, i64 %1, i64 %2, i64 %3) #1 section \"helpers\" {
```